repair_tests/deprecated_repair_test.py

```diff
@@ -85,6 +85,9 @@ def test_force_repair_range_async_1(self):
                                    Collection<String> hosts, boolean fullRepair,
                                    String... columnFamilies)
         """
+        self.fixture_dtest_setup.ignore_log_patterns = [
+            'Nothing to repair for'
+        ]
```

Comment:
Can you provide a bit of info when we see this case and why it is ok to ignore it, please?

Reply:
This is the log we generate when we skip repair due to no data (or no tables, etc.); this test does *not* write data, so skipping repair is the expected behavior

Reply:
added docs explaining why; these tests do repairs for a token range, but doesn't put in effort to make sure that there are partitions in that range, so the repair aborts.

The test is checking logging, which is done after we parse arguments and print them... if repair doesn't happen these tests don't really care.